std/pdb.zig

```diff
@@ -632,7 +632,7 @@ const MsfStream = struct {
     }
 
     fn read(self: *MsfStream, buffer: []u8) !usize {
-        var block_id = self.pos / self.block_size;
+        var block_id = @intCast(usize, self.pos / self.block_size);
         var block = self.blocks[block_id];
```

Comment:
This is a very strange type error, it was saying that it expected `block_id` to be a `usize` on line 636; I didn't understand why this suddenly started happening, so I just added the `@intCast` to fix it temporarily.

Reply:
This fix is also necessary for 32-bit windows most likely.